src/partials/contacts.html

```diff
@@ -0,0 +1,80 @@
+      <main>
+      <section class="contact">
+        <h2>Contacts</h2>
+        <!--VISUALLY HIDDEN-->
+        <ul class="contacts-list">
+          <li class="contacts-main-item">
+            <p class="contacts-place-type">CAFE</p>
+            <p class="contacts-city">
+              Chicago
+              <a href="#" class="contacts-adress"
+                >10340 Ridgeland Ave, Chicago Ridge, IL 60415, USA</a
+              >
+              <hr> <!--znaczniki hr zeby troche to oddzielic pewnie pozniej do wywalenia-->
+              <!--link do gmaps w #-->
+            </p>
```

Comment:
Zamykasz tag <p> po <hr>. Jest to błąd. Usuń wszystkie hr, albo przenieś /p nad nie.